.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,42 +1,178 @@
+### Privacy-First (IMMUTABLE)
+- [ ] ✅ No PII leaks (emails, names, SSNs sanitized in exports/logs)
+- [ ] ✅ localStorage-first (no server required for core features)
+- [ ] ✅ Explicit consent for any server features
+- [ ] N/A - No privacy-related changes
+
+### Accessibility-First (IMMUTABLE)
+- [ ] ✅ WCAG 2.2 AA compliance verified
+- [ ] ✅ Keyboard navigation tested (see Manual Testing below)
+- [ ] ✅ Screen reader tested (see Manual Testing below)
+- [ ] ✅ Color contrast verified (4.5:1 text, 3:1 UI)
+- [ ] N/A - No UI changes
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)